mesa/examples/advanced/wolf_sheep/Readme.md

```diff
@@ -35,7 +35,7 @@ To run the model interactively, run ``mesa runserver`` in this directory. e.g.
     $ mesa runserver
 ```
 
-Then open your browser to [http://127.0.0.1:8521/](http://127.0.0.1:8521/) and press Reset, then Run.
+Then open your browser to [http://127.0.0.1:8765/](http://127.0.0.1:8765/) and press Reset, then Run.
```

Comment:
Line 17-38 are no longer relevant, these are artifacts of Mesa 2, could you update this so it matches the Solara approach for Mesa 3 (e.g. `solara run app.py`)